src/index.js

```diff
@@ -96,6 +96,18 @@ window.apps = apps;
 
   console.log(`Loaded apps marquee with ${apps.length} items.`);
 
+  const latestIPADownload = document.querySelector('#latest-ipa-download');
+  const latestRelease = await (await fetch('https://api.github.com/repos/SideStore/SideStore/releases/latest')).json();
+  if ('assets' in latestRelease)
+    for (const asset of latestRelease.assets) {
+      if (asset.name.endsWith('.ipa')) {
+        latestIPADownload.href = asset.browser_download_url;
+        console.log('Found latest release and .ipa', asset, latestRelease);
```

Comment:
I will also implement some loading indicators on this, for low-bandwidth connections.

Reply:
These changes are not needed and outdated, since we can just use a direct latest download link.